src/credential/mod.rs

```diff
@@ -129,22 +425,36 @@ pub enum VcdmVersion {
 
 #[derive(Debug, uniffi::Error, thiserror::Error)]
 pub enum CredentialEncodingError {
-    #[error(transparent)]
-    MsoMdoc(#[from] MdocEncodingError),
-    #[error(transparent)]
-    JsonVc(#[from] JsonVcEncodingError),
```

Comment:
NOTE: I had to remove `thiserror::Error` macro attributes due to limitations in Kotlin generated code.

There may be another work around that I am not aware of, can add back if other know how to satisfy android compilation using the generated kotlin code.

Reply:
Check using `#[uniffi(flat_error)]` to resolve uniffi bindgen issues.

Reply:
> NOTE: I had to remove thiserror::Error macro attributes due to limitations in Kotlin generated code.

Can you explain? I can't see any issues with it when I am try to use the current Exception in Kotlin.

Reply:
I was able to confirm that it does work for using the `#[from]`, and was not able to reproduce the original error I saw when compiling Kotlin.

I've re-added the macros and did not need to use `#[uniffi(flat_error)]`